mmstudio/src/main/java/org/micromanager/internal/PropertyEditor.java

```diff
@@ -216,14 +217,24 @@ public PropertyEditorTableData(Studio studio, String groupName, String presetNam
 
       @Override
       public void setValueAt(Object value, int row, int col) {
-         PropertyItem item = propListVisible_.get(row);
-         studio_.logs().logMessage("Setting value " + value + " at row " + row);
-         if (col == propertyValueColumn_) {
-            setValueInCore(item, value);
+         try {
+            PropertyItem item = propListVisible_.get(row);
+            if (core_.getDeviceType(item.device) == DeviceType.CameraDevice) {
+               studio_.live().setSuspended(true);
+            }
+            studio_.logs().logMessage("Setting value " + value + " at row " + row);
+            if (col == propertyValueColumn_) {
```

Comment:
I think this check for `col == propertyValueColumn_` should be done at the top of the method (and just return if false). I don't know if this method is ever called with a different column (it probably shouldn't), but it doesn't make sense to pause and resume Live if no change is being made. Also, that way the logging will match reality.